lib/travis/api/v3/models/json_sync.rb

```diff
@@ -5,7 +5,7 @@ module Models::JsonSync
     def sync(parent, attr)
       @parent, @attr = parent, attr
       @sync = -> do
-        previous = @parent.attributes[@attr.to_s] || {}
+        previous = @parent.send(:"#{@attr}") || {}
```

Comment:
BTW I don't know why we were doing this before, you can send `@attr` as it is, it doesn't matter if it's a symbol or a string:

```
irb(main):001:0> "hey".send("upcase")
=> "HEY"
irb(main):002:0> "hey".send(:upcase)
=> "HEY"
```